cmake/modules/RootBuildOptions.cmake

```
@@ -382,13 +382,19 @@ foreach(opt afdsmgrd afs bonjour castor chirp geocad glite globus hdfs ios
   endif()
 endforeach()
 
-#---Deprecated options---------------------------------------------------------
-foreach(opt memstat vmc python)
+#---Deprecated options------------------------------------------------------------------------
+foreach(opt memstat vmc)
   if(${opt})
     message(DEPRECATION ">>> Option '${opt}' is deprecated and will be removed in the next release of ROOT. Please contact root-dev@cern.ch should you still need it.")
   endif()
 endforeach()
 
+#---Replaced options--------------------------------------------------------------------------
+if(python)
+  message(DEPRECATION ">>> Option 'python' was replaced by 'pyroot' option, that will be enabled for you instead. To disable 'python' option, please just use 'pyroot' option directly.")
```

Here, `-Dpython=On` was specified (because `if(python)`). So talking about how to disable it seems not relevant. Maybe:

```suggestion
  message(DEPRECATION ">>> Please use the equivalent option 'pyroot' instead of 'python'; the latter will be removed in v6.22.")
```

Can we emit a warning also in the case where people specify `-Dpython=Off`?